spec/person_spec.rb

```diff
@@ -0,0 +1,11 @@
+$LOAD_PATH << '.'
+require 'spec_helper'
+
+describe Person do
+  it 'Create a new person' do
+    @new_person = Person.new(19, 'Levy')
+    expect(@new_person.age).to eq(19)
+    expect(@new_person.name).to eq('Levy')
+    expect(@new_person.parent_permission).to eq(true)
+  end
```

Comment:
- [ ] Same issue as above. Besides the initialize method, the `Person` class has other four functions we can cover with tests. Please, let's address this.